libraries/dotnet/EnergyOrigin.ActivityLog/EnergyOrigin.ActivityLog/ActivityLogExtensions.cs

```diff
@@ -20,7 +20,7 @@ public static IServiceCollection AddActivityLog(this IServiceCollection services
     public static RouteHandlerBuilder UseActivityLog(this IEndpointRouteBuilder builder)
     {
         return builder.MapPost(
-            "/activity-log",
+            "api/activity-log",
```

Comment:
Explain to me what this does. Is the library itself hosting an endpoint now?

And is that endpoint "api/activity"?

Or is that endpoint "api/{name-of-entity}/activity-log"?

Reply:
The package is not hosting an API endpoint, but adding support for the other API's can host it.

So the endpoint is: "api/activity-log" <-- activity-log is name of the resource. Just forgot to add api part to follow other API endpoints.

Reply:
The first version of nuget package was merged because we wanted a build of the nuget package, so we could work on implementing logic in the different submodules / sub systems, while doing final work on nuget package.